Packs/FWToCDLMonitoring/Scripts/FWToCDLMonitoring/FWToCDLMonitoring.yml

```diff
@@ -0,0 +1,36 @@
+args:
+- description: Comma separated list of FW serial numbers to monitor.
+  name: fw_serials
+- description: Panorama integration name to retrieve FW list
+  name: panorama
+comment: "Verify that all firewalls successfully pushed logs to the Cortex Data Lake\
+  \ for the last 12 hours. It's an easy way to do monitoring of the FW connection\
+  \ to CDL.\nYou can use either a manual list of FW serials or a Panorama integration\
+  \ to get the list of equipment to monitor. "
+commonfields:
+  id: FW-to-CDL-monitoring
+  version: -1
+dependson:
+  must:
+  - Panorama|||panorama
+  - Cortex Data Lake|||cdl-query-traffic-logs
+dockerimage: demisto/python3:3.9.2.17246
+enabled: true
+name: FW-to-CDL-monitoring
+outputs:
+- contextPath: monitoring_result
```

Comment:
According to the code, this is not the context path.
Please fix it.

Reply:
Fixed

Reply:
Please take a look at the following article to understand the context paths: https://xsoar.pan.dev/docs/integrations/context-and-outputs

Reply:
I had a close look at the doc and I modified the command_results to match this example: https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-data-that-has-multiple-unique-identifier-fields